styles.css

```diff
@@ -20,12 +42,10 @@
     height: var(--menu-bar-height);
     position: absolute;
 }
-
 #menu::before {
     left: 0;
     top: -20px;
 }
-
 #menu::after {
     left: 0;
     top: 20px;
```

Comment:
ziet er prima verder uit, laat even weten of hij nu werkt.... groet maarten

Reply:
Ik zal dit nog even checken en dan hoor je van mij!

Reply:
Ik heb de rotation after meer in degrees (135).  Weet jij wanneer het een CSS conventie is (dus dat het op een bepaalde wijze dient gedefinieert te worden of wanneer er ook meerdere wegen naar Rome leiden. Dit is bij het checken van elkaars huiswerk soms mijn vraag. Wellicht goed voor terugkoppeling in les.

Reply:
Dus niet 135 degrees maar the other way around -45 Nu begrijp ik hem!